maven_iuvs/echelle.py

```diff
@@ -2086,10 +2174,11 @@ def lineshape_model(params, wavelength_data, binedges, theCLSF, BU_bg, fit_IPH=F
 
     # Do the IPH fit, if requested
     if fit_IPH:
-        total_brightness_IPH = params[-2]
-        central_wavelength_IPH = params[-1]
-        interpolated_CLSF_IPH = interpolate_CLSF(central_wavelength_IPH, binedges, theCLSF)
-        normalized_line_shape_IPH = np.diff(interpolated_CLSF_IPH) # Unitless
+        total_brightness_IPH = params[-3] 
+        central_wavelength_IPH = params[-2]  # TODO: Can calculate based on spacecraft ephemeris and pass in for the starting guess/requirement.
+        width_IPH = params[-1]
+        CDF_IPH = sp.stats.norm.cdf( (binedges - central_wavelength_IPH) / width_IPH ) #  TODO: Should also be able to calculate this, which is the width of the Gaussian.
+        normalized_line_shape_IPH = np.diff(CDF_IPH)
         IPH_fit = total_brightness_IPH * normalized_line_shape_IPH
```

Comment:
This seems like it should work to me--- maybe you need some additional heuristics to guess good initial values based on the observed spectrum?

Reply:
To respond to above comment, not even Dynesty was able to fit these for some reason